temporalio/client.py

```diff
@@ -456,6 +460,7 @@ async def start_workflow(
         rpc_timeout: Optional[timedelta] = None,
         request_eager_start: bool = False,
         stack_level: int = 2,
+        priority: Optional[temporalio.common.Priority] = None,
```

Comment:
Also probably need this on `ScheduleActionStartWorkflow` unless we're intentionally leaving off that for now